src/main/java/workout/workoutsession/WorkoutSession.java

```diff
@@ -44,16 +43,19 @@ public void workoutSessionStart() {
         this.cl = new CommandLib();
         cl.initWorkoutSessionCL();
         Scanner in = new Scanner(System.in);
+
         try {
             storage.readFileContents(filePath, exerciseList);
         } catch (FileNotFoundException e) {
             WorkoutSessionUi.printError();
         }
+
         while (!endWorkoutSession[0]) {
             try {
+                System.out.print("Workout Session >>>>> ");
                 workoutSessionProcessCommand(in.nextLine().trim());
             } catch (NullPointerException e) {
-                e.printStackTrace();
+                System.out.println("Please enter something.");
```

Comment:
Perhaps can implement the UI printing here also

Reply:
will do